src/main/java/org/myrobotlab/service/abstracts/AbstractSpeechSynthesis.java

```diff
@@ -537,31 +543,31 @@ public Long publishGenerationTime(Long timeMs) {
    * @throws UnsupportedEncodingException 
    */
   public AudioData process(AudioData audioData, String speak, boolean block) {
- 
-      try {
 
-        long generateStartTs = System.currentTimeMillis();
-        utterances.put(audioData, speak);
-        
-        if (!audioData.isValid()) {
-          log.info("try generating audio data [{}] from [{}]", audioData, speak);          
-          generateAudioData(audioData, speak);          
-        }
-        
-        invoke("publishGenerationTime", System.currentTimeMillis() - generateStartTs);
+    try {
 
-        if (!audioData.isValid()) {
-          log.error("speech service could not generate audio data [{}]", audioData);
-          return audioData;
-        }
-                
-        play(audioData, block);
-      } catch (Exception e) {
-        log.error("could not generate audio", e);
-        error("%s %s", e.getClass().getSimpleName(), e.getMessage());
+      long generateStartTs = System.currentTimeMillis();
+      utterances.put(audioData, speak);
+
+      if (!audioData.isValid()) {
+        log.info("try generating audio data [{}] from [{}]", audioData, speak);
+        generateAudioData(audioData, speak);
       }
 
-      return audioData;
+      invoke("publishGenerationTime", System.currentTimeMillis() - generateStartTs);
+
+      if (!audioData.isValid()) {
+        log.error("speech service could not generate audio data [{}]", audioData);
+        return audioData;
+      }
+
+      play(audioData, block);
+    } catch (Exception e) {
+      log.error("could not generate audio", e);
+      error("%s %s", e.getClass().getSimpleName(), e.getMessage());
+    }
+
+    return audioData;
   }
 
```

Comment:
Were there changes here ? - the git compare seems to freak out

Reply:
no change here, except the formatter